### PR TITLE
Release 1.37.0

### PR DESCRIPTION
**Pull request by johnstcn**

### CODE REVIEW

File: changelog/1.37.0.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### Breaking changes ❗
+
+- infra: Workspace statistics were previously stored in a table
+  `environment_stats`. This table is removed in this release. Downgrading from
+  this version may require manual steps.
+- cli: `coder ssh` and `coder tunnel` invocations from older versions of the CLI
+  may fail with the following error:
+
+  ```console
+  fatal: get ICE servers: decode response body: json: cannot unmarshal string into Go struct field ICEServer.data.credentialType of type webrtc.ICECredentialType
```
Comment (johnstcn) [inline changelog/1.37.0.md:15]: @johnstcn Need to shorten this line!

File: changelog/1.37.0.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  ```
+
+  Solution: upgrade CLI to `1.37.1`.
```
Comment (johnstcn) [inline changelog/1.37.0.md:18]:
```suggestion
  Solution: upgrade CLI to `1.37.0`.
```

File: changelog/1.37.0.md (hunk 1 above)
Comment (johnstcn) [inline changelog/1.37.0.md:15]:
```suggestion
cannot unmarshal string into Go struct field ICEServer.data.credentialType of type webrtc.ICECredentialType
```

**Review by spikecurtis — APPROVED** [review]

File: changelog/1.37.0.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- infra: Workspace statistics were previously stored in a table
+  `environment_stats`. This table is removed in this release. Downgrading from
+  this version to an older version may require manual steps.
+- cli: `coder ssh` and `coder tunnel` invocations from older versions of the CLI
+  may fail with the following error:
+
+  ```console
+  cannot unmarshal string into Go struct field ICEServer.data.credentialType of
+  type webrtc.ICECredentialType
+  ```
```
Comment (spikecurtis) [inline changelog/1.37.0.md:17]: is this still broken? I'm able to ssh just fine with 1.36.0 CLI to the RC

**Review by code-asher — APPROVED** [review]

File: changelog/1.37.0.md (hunk 3 above)
Comment (johnstcn) [reply changelog/1.37.0.md:17]: Discussed: should be OK for this release as the breaking JSON marshalling change isn't in the version of `pion/webrtc` used in this release.

**Review by spikecurtis — APPROVED** [review]

**Pull request closed without merging**